TTS/tts/datasets/dataset.py

```diff
@@ -34,6 +35,13 @@ def noise_augment_audio(wav):
     return wav + (1.0 / 32768.0) * np.random.rand(*wav.shape)
 
 
+def string2filename(string):
+    # return re.sub('[^\w_.)( -]', '_', string)
```

Comment:
do we need this line?

Reply:
We need to convert the "audio_unique_name" that is something like "ljspeech#wavs/LJ-0001" to a valid file name. I found two good ways to do that. The first uses regular expression and the second is with base64 which looks like the better solution. I kept the re version on the code just in case. Do you want me to remove this comment line?

Reply:
yeah I was about the comment line

Reply:
oh yeah, I removed it :).